tests/5.0/target/test_target_parallel_linear.c

```diff
@@ -46,8 +46,7 @@ int main() {
   OMPVV_TEST_OFFLOADING;
   int TotGpus = omp_get_num_devices();
   int errors = 0;
-  for (int gpu = 0; gpu < TotGpus; ++gpu) {
-    errors = 0;
+  for (int gpu = 0; gpu <= TotGpus; ++gpu) {
```

Comment:
Why <= ?

Reply:
No strong reason but as written above:
> Looking at the testcase, I saw that it only run on non-initial devices - but the test function should also work on the host; hence, it is now tested there as well.

If there is no code reason that something only works on non-host devices, I think it can also be run on the host – which is especially useful when one runs it on a system lacking offload devices.

It then at least tests whether `parallel for device(gpu) linear(val:2)` does somewhat sensible on the host (in a host-fallback target region).